netplugin/agent/agent.go

```diff
@@ -79,6 +80,9 @@ func NewAgent(pluginConfig *plugin.Config) *Agent {
 		log.Fatalf("Failed to initialize the plugin. Error: %s", err)
 	}
 
+	// init mesos plugin
+	mesosplugin.InitPlugin(netPlugin)
```

Comment:
please see if we can specify multiple plugin modes like "docker,mesos"
That way we can initialize only some mgmtfns..